composer.json

```diff
@@ -12,5 +12,9 @@
         "php": "^7.4",
         "phpmailer/phpmailer": "^6.1",
         "vanilla/nbbc": "^2.0"
+    },
+    "require-dev": {
+        "phpunir/phpunit": "9.1.4",
```

Comment:
Is `phpunir` a typo? Looks like a weird package that's a wrapper around `phpunit`? https://github.com/MaurizioMoreo/test

Reply:
Wow, very strange -- I hadn't even noticed that. I let autocomplete work itself out, and I thought I saw `phpunit` :D I'm gonna make sure to change this to the official distro.